backend/src/config/index.ts

```diff
@@ -26,6 +26,7 @@ export const getJwtSignupLifetime = async () => (await client.getSecret("JWT_SIG
 export const getJwtProviderAuthSecret = async () => (await client.getSecret("JWT_PROVIDER_AUTH_SECRET")).secretValue;
 export const getJwtProviderAuthLifetime = async () => (await client.getSecret("JWT_PROVIDER_AUTH_LIFETIME")).secretValue || "15m";
 export const getJwtSignupSecret = async () => (await client.getSecret("JWT_SIGNUP_SECRET")).secretValue;
+export const getJwtServiceTokenSecret = async () => (await client.getSecret("JWT_SERVICE_TOKEN_SECRET")).secretValue;
```

Comment:
idea: what if we set all of the jwt secrets to the single encryption key? Feel like we have a lot of JWT secrets. If one JWT gets compromised, surely the others would too?

Reply:
I might have a better idea for this part. I need to go over the security implications first - We shouldn’t just set it to the encryption key at all; this would have severe implications on parts like MFA currently that’s contingent on differentiation between the secret for that temporary token and regular JWT token.

An option would be to use the encryption key as a seed to generate 6 other secrets sequentially.

Another option might be to programmatically generate the JWT secrets on the backend randomly and encrypt them under the encryption key.